src/CMakeLists.txt

```diff
@@ -34,6 +34,14 @@ if(NOT (BUILD_CLIENT OR BUILD_SERVER))
 endif()
 
 
+option(ENABLE_TOUCH "Enable Touchscreen support" FALSE)
+
+if(ENABLE_TOUCH)
+	add_definitions(-DHAVE_TOUCHSCREENGUI)
+	find_library(XINPUT_LIBRARY Xi REQUIRED)
```

Comment:
why is this suddenly required?

Reply:
I moved it to the correct section.

Reply:
I meant this one: https://github.com/minetest/minetest/blob/ee1bda90edc5a051b059fe61959836e062be37f4/src/CMakeLists.txt#L338-L363

Reply:
Done!